crl_util.go

```diff
@@ -1,175 +1,16 @@
 package keyfactor
 
-import (
```

Comment:
I think this functionality was moved to `path_certs.go` should we just delete this file?